src/library/scala/collection/ArrayOps.scala

```diff
@@ -407,6 +407,21 @@ final class ArrayOps[A](val xs: Array[A]) extends AnyVal {
     (res1.result(), res2.result())
   }
 
+  /** A pair of, first, all elements mapped by `f` to a `Left` and, second, all elements mapped by `f` to a `Right`. */
+  def partitionMap[A1: ClassTag, A2: ClassTag](f: A => Either[A1, A2]): (Array[A1], Array[A2]) = {
```

Comment:
How about adding some description of the parameter and the returning result?

Reply:
For some reason, `ArrayOps#partition` only has a subset of the documentation of `Iterable#partition` so I assumed it was a convention and did the same for `partitionMap` (only copying the first line). Should these docs actually be copy-pasted from the `Iterable` ones?

Reply:
The docs should generally be the same (but adapted for use with arrays rather than different collection types). Some old methods may not have the full doc comments. They should be updated.